Dockerfile

```diff
@@ -67,8 +67,8 @@ ADD https://nodejs.org/dist/v8.11.3/node-v8.11.3-linux-x64.tar.gz /builds/worker
 ADD https://dl.google.com/android/android-sdk_r24.3.4-linux.tgz /builds/worker/Downloads
 #COPY downloads/android-sdk_r24.3.4-linux.tgz /builds/worker/Downloads
 
-ADD https://github.com/taskcluster/taskcluster-worker/releases/download/v0.1.20/taskcluster-worker-0.1.20-linux-amd64 /usr/local/bin/taskcluster-worker
-#COPY downloads/taskcluster-worker-0.1.20-linux-amd64 /usr/local/bin/taskcluster-worker
+ADD https://github.com/taskcluster/generic-worker/releases/download/v13.0.4/generic-worker-linux-amd64 /usr/local/bin/generic-worker
+#COPY downloads/generic-worker-linux-amd64 /usr/local/bin/generic-worker
```

Comment:
Do you think it is worth it to go ahead and install https://github.com/taskcluster/livelog ?

Reply:
Yeah, I did it in another branch (https://github.com/bclary/mozilla-bitbar-docker/compare/master...aerickson:gw_v2_1), will port it over.